tests/test_main.py

```diff
@@ -119,14 +120,21 @@ def test_make_tables_handles_errors(self) -> None:
     def test_make_generators(self) -> None:
         """Test the make-generators sub-command."""
         with patch("sqlsynthgen.main.make_generators_from_tables") as mock_make:
+            conf_path = "tests/examples/generator_conf.yaml"
```

Comment:
Not a suggestion but I thought I'd mention that I found out this week that you can call `.open()` on pathlib objects so 

```suggestion
            conf_path = pathlib.Path("...")
            with conf_path.open():
                ...
```

will also work.

Reply:
👍